### ISSUE TITLE
Backlog

### ISSUE THREAD

**Issue by balsama** [acquia/lightning#180]
Infrastructure
- [x] move to selenium
- [x] test new packagist
  - [x] Can we have two packagist? <-- Yes, but only if they don't define the same packages. If they _do_ define the same package, the last packagist to define that package wins, and only its versions are installable.
  - [x] ~~Does it build dependencies~~ <--It does not. Blocked on this
- [x] Don't test so much
- [ ] Improve testing - use lightning project to build test
- [ ] Move away from Phing, use scripts in Lightning Project's composer.json to do build tasks.
- [ ] Start using asset-packagist instead of NPM and Bower
- [ ] Split the tests into suites separated by component

Preview
- [ ] Can't truly resolve conflicts. See [comment on docs here](https://docs.google.com/document/d/10cBoY74LrjVMWSolMwarLNwes4usepHkE-kuE_y7Lk0/edit?disco=AAAAAvYNAvU)
- [x] Authors don't replicate. [Related code in replication](https://github.com/relaxedws/drupal-replication/blob/8.x-1.x/src/Normalizer/EntityReferenceItemNormalizer.php#L38)
- [ ] PHP warning
- [ ] 2819317 Class not found <-- According to the issue, this causes a consistent test failure. Our tests have not been failing, so this is postponed until further notice.
- [ ] URL aliases - prefixes only work for nodes
- [x] URL aliases - overrides don't work
- [ ] Blocks list page lists thing twice
- [x] Core file system issue needs tests
- [ ] Config entities need to be able to be isolated by workspace

Workflow
- [ ] Migration path for content moderation (joe purcell?)
- [ ] Add notifications to state transitions
- [x] Add Diff module and figure out how best to integrate it into Lightning

Media
- [x] Twitter SVGs
- [x] Videos are not constrained within ckeditor (core issue + lightning)
- [ ] Make Entity Embed entities inline-able

Layout
- [ ] Blocks in layouts should be tied to a revision
- [ ] Move IPE to tray without outside in interfering with current system
- [ ] Wizard sucks - document specific steps
- [x] Panelize all core entities (panelizer issues exist already)
- [ ] After switching to new layout, lightning should more clearly show where the regions are
- [x] ["Revert to default" destroys published version of page](https://www.drupal.org/node/2754649)
